packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -350,14 +350,23 @@ export const Audio = () => {
           setListenLoggedForTrack(false)
         )
       }
-      global.progress = progress
+
+      if (!isCasting) {
+        // If we aren't casting, update the progress
+        global.progress = progress
+      } else {
+        // If we are casting, only update the seekableDuration
+        // The currentTime is set via the effect in GoogleCast.tsx
+        global.progress.seekableDuration = progress.seekableDuration
```

Comment:
Do we need to also set this for when we are not casting?

Reply:
Yes, in that case we update the entire progress object. Which includes seekableDuration